api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -661,7 +574,7 @@ public DbUser syncComplianceTrainingStatusV2(DbUser dbUser, Agent agent)
               accessModuleService.updateCompletionTime(
                   dbUser, accessModuleName, timestamp.orElse(null)));
 
-      return updateUserAccessTiers(dbUser, agent);
+      return accessTierSyncService.updateUserAccessTiers(dbUser, agent);
```

Comment:
TODO: many of these calls should be delegated to updateCompletionTime() / updateBypassTime()